brigade/templates/resources_videos.html

```diff
@@ -0,0 +1,95 @@
+{% extends "base.html" %}
+{% block title %}Videos{% endblock %}
+{% block description %}Recordings from events within the Brigade Network{% endblock %}
+{% block page_id %}videos{% endblock %}
+
+{# Conditionals based on what type of page is rendering #}
+{% if topic %}
+{% else %}
+{% endif %}
+
+{% block content %}
+<header id="intro" class="page-header slab slab-dark-blue">
+  <div class="grid-box">
+    <h1 class="title">Videos</h1>
+    <div class="subtitle">Recordings from events within the Brigade Network</div>
+  </div>
+</header>
+
+{% if not topic %}
+<section class="slab slab--compact">
+  <div class="grid-box">
+    <div class="width-one-half">
+      <div class="alert">
+        <div class="alert__icon">
+          <i class="fas fa-info-circle"></i>
+        </div>
+        <div class="alert__content">
+          Check out <a href="https://www.youtube.com/user/CodeforAmerica/videos" target="_blank">Code for America's YouTube</a> for more videos.
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+{% endif %}
+
+<section class="slab slab--compact">
+  <div class="grid-box">
+    {% if topic %}
+    <p><a href="{{ url_for('.resources_videos') }}">&larr; Back to all videos</a></p>
+    {% endif %}
+
+    <h2 class="slab__title">
+      {% if topic %}
+      Latest Videos Tagged "{{ topic }}"
+      {% else %}
+      Latest Videos
+      {% endif %}
+    </h2>
+  </div>
+
+  <div class="grid-box">
+    <div class="width-one-half">
+      <p class="slab__subtitle">
+        {% if topic %}
+        {% else %}
+        Looking for a recording of an event or gathering in the Brigade network? You've come to the right place.
+        {% endif %}
+      </p>
+    </div>
+  </div>
+
+  <div class="grid-box">
+    {% for video in videos[0:3] %}
+    <div class="width-one-third">
+      <div class="card card--multimedia">
+        <div class="card__body">
+          <iframe width="100%" height="240" src="{{ video["id"] | youtube_link(start=video.get("start"), embed=True) }}" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+          <h2 class="card--multimedia__title">
+            <a href="{{ video["id"] | youtube_link(start=video.get("start")) }}">
+              {{ video["title"] }}
+            </a>
+          </h2>
+          <p class="card--multimedia__metadata">{{ video["date"] }}</p>
+
+          <p>{{ video["description"] }}</p>
+        </div>
+
+        <div class="card__footer">
+          <i class="fas fa-tags"></i>
+          Posted in {{ video["topics"] | map('link_to_video_topic') | join_list }}
+        </div>
+      </div>
+    </div>
+    {% endfor %}
+  </div>
+</section>
+
+<section class="slab">
+  <div class="grid-box">
+    <a href="{{ more_url }}" target="_blank" class="button button-primary">View more on YouTube</a>
```

Comment:
Another minor design nitpick: I wonder if the this button should be centered? On the left it feels a little _floaty_.